sig-contributor-experience/charter.md

```diff
@@ -1,58 +1,143 @@
+- We are not the [code of conduct committee] and therefore do not control incident management reporting or decisions; however, our moderation guidelines allow us to act swiftly if there is a clear violation of terms of either our code of conduct or one of our supported platforms terms of service. If there is an action that the committee needs to take that involves one of these platforms (example: the removal of someone from GitHub), we will carry that out if none of the committee members have access.
+- Communication platforms that are out of our scope for maintenance and support but we may still have some influence:
+    - [r/kubernetes]
+    - [@kubernetesio] twitter account
+    - [kubernetes blog]
+
+## Roles and Organization Management
+
+This sig adheres to the Roles and Organization Management outlined in [sig-governance]
+and opts-in to updates and modifications to [sig-governance].
+
+
+### Additional responsibilities of Chairs
+
+Chairs SHOULD plan at least one face to face Contributor Experience meeting per year
+
+### Additional responsibilities of Tech Leads
+
+Ensuring that technical changes from subprojects follow the process change communication guidelines listed above.
+
+### Deviations from sig-governance
+Six months after this charter is first ratified, it MUST be reviewed and re-approved by the SIG in order to evaluate the assumptions made in its initial drafting.
+
+### Subproject Creation
+Chairs and Technical Leads
+
+[sig-governance]: https://git.k8s.io/community/committee-steering/governance/sig-governance.md
+[Kubernetes Charter README]: https://git.k8s.io/community/committee-steering/governance/README.md
+[lazy consensus]: http://en.osswiki.info/concepts/lazy_consensus
+[Contributor Experience Special Interest Group]: https://groups.google.com/forum/#!forum/kubernetes-sig-contribex
+[kubernetes-dev@]: https://groups.google.com/forum/#!forum/kubernetes-dev
+[@kubernetesio]: https://www.twitter.com/kubernetesio
+[r/kubernetes]: https://kubernetes.reddit.com
+[Google Summer of Code]: https://git.k8s.io/community/mentoring/google-summer-of-code.md
+[Outreachy]: https://git.k8s.io/community/mentoring/outreachy.md
+[Meet Our Contributors]:  https://git.k8s.io/community/mentoring/meet-our-contributors.md
+[Group Mentoring - WIP]:  https://git.k8s.io/community/mentoring/group-mentoring.md
+[The 1:1 Hour - WIP]: https://git.k8s.io/community/mentoring/the1-on-1hour.md
+[kubernetes/community]: https://git.k8s.io/community/
+[Contributor Summit(s)]: https://git.k8s.io/community/events/2018/12-contributor-summit
+[contributor summits]: https://git.k8s.io/community/events/2018/12-contributor-summit
+[DevStats]: https://k8s.cncf.devstats.io
+[kubernetes-sig-contribex@]: https://groups.google.com/forum/#!forum/kubernetes-sig-contribex
+[kubernetes blog]: https://www.kubernetes.io/blog
+[GitHub]: https://git.k8s.io/community/github-management
```

Comment:
nit: let's call this github-management instead of just `GitHub`? :grimacing:

Reply:
fixed